Qt.py

```diff
@@ -893,9 +893,14 @@ def load(self, uifile, *args, **kwargs):
                 etree = ElementTree()
                 etree.parse(uifile)
 
-                return Qt._QtUiTools.QUiLoader.load(
+                widget = Qt._QtUiTools.QUiLoader.load(
                     self, uifile, *args, **kwargs)
 
+                # Prevent RuntimeError in Nuke 9.0v9 with PySide 1.0.9
+                Qt.QtCompat._loadUi_temp = widget.parentWidget()
```

Comment:
This is most likely not Nuke-specific. I even got the workaround from the [PySide bug tracker](https://bugreports.qt.io/browse/PYSIDE-10).

If we were to repro this outside of Nuke we'd better try it in PySide 1.0.9. Not sure how I could most easily install that old version. Perhaps the only solution is to compile it from scratch:
https://github.com/pyside/PySide/releases/tag/1.0.9

:dizzy_face:

Any other ideas?